charts/kyverno-policies/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+type: application
+name: cray-kyverno-policies-upstream
+version: v1.0.0
+appVersion: v1.6.2
+icon: https://github.com/kyverno/kyverno/raw/main/img/logo.png
+description: Kubernetes Pod Security Standards implemented as Kyverno policies
+keywords:
+  - kubernetes
+  - policy agent
+  - validating webhook
+  - admissions controller
+dependencies:
+  - name: kyverno-policies
+    repository: https://kyverno.github.io/kyverno/
+    version: v2.3.2
+home: https://github.com/Cray-HPE/cray-kyverno-policies-upstream
+sources:
+  - https://github.com/kyverno/policies
+maintainers:
+  - name: pradeepg
+    url: https://kyverno.io/
```

Comment:
You are not the maintainer of the upstream content. If you are going to include this, use the GitHub URL (Cray/HPE) to the repo.